app/screens/SearchLibrary/components/Header/index.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { ImageBackground, TextInput, View, Image, TouchableOpacity } from 'react-native';
+import headerImg from '@assets/bc_nav_bar.png';
+
+import cancelIcon from './assets/cancel.png';
+import searchIcon from './assets/ic_search.png';
+import { styles } from './styles';
+
+class Header extends Component {
+  state = {
+    searchText: ''
+  };
+
+  handleOnChangeText = text => {
+    this.setState({ searchText: text });
+    const onChangeText = this.props.navigation.getParam('onChangeText');
```

Comment:
beware if you don't receive anything because the component is not fully rendered